File: RecorridoInverso/RecorridoInverso.java
public class RecorridoInverso
{
    public static void main(String[] args)
    {
        int arr[]={40, 50, 60, 70, 80, 90};
        int arreglo=arr.length;
        System.out.println("Los elementos del array son: ");
        for(int idx = arreglo - 1; idx >= 0; idx--)
        {
            System.out.print(arr[idx]+" ");
        }
    }
}

File: RecorridoInverso/RecorridoInverso.js
let arr=[40, 50, 60, 70, 80, 90];
let arreglo=" ";

for (let idx = arr.length - 1; idx >= 0; idx--)//ultimo elemto del arregli menos uno continua hasta que sea mayor o igual a cero mientas va restando uno
    {
        arreglo += arr[idx] + " ";
    }
    console.log("Los elementos del array son: " + arreglo);